go/interpreter/lfvm/interpreter.go

```diff
@@ -243,30 +243,12 @@ func steps(c *context, oneStepOnly bool) {
 
 		op := c.code[c.pc].opcode
 
-		// JUMP_TO is an LFVM specific operation that has no gas costs nor stack usage.
-		if c.code[c.pc].opcode == JUMP_TO {
-			c.pc = int32(c.code[c.pc].arg)
-			op = c.code[c.pc].opcode
-		}
-
-		// Catch invalid op-codes here, to avoid the need to check them at other places multiple times.
-		if op >= NUM_EXECUTABLE_OPCODES {
-			c.signalError()
```

Comment:
this is something we could improve next:



The new invalid instruction (that replaces the deleted code), does something slightly different now, this has been noticed and the test has been rewritten. 
```
default:
			c.status = statusInvalidInstruction
			return
		}
```


There is a different style to report errors, signalError | c.status assign.
- I am not a big fan of signalError, as it seems to do something else but an assignment, maybe setError would be better.
- on the other hand, we may not need this fine granularity of errors, as mentioned here: https://github.com/Fantom-foundation/Tosca/blob/c9b04766a92b31bcab146f1b0ba64e459086384e/go/interpreter/lfvm/interpreter.go#L167


It is true that some other testing tools may be expecting some of these codes, but the invalid instruction is not one of them because it was triggering the deleted if evey time, therefore producing generic error status.

Reply:
The function `signalError()` should probably be renamed to `abort()`. About the different error cases -- I think the fine granularity is nowhere utilized and we can generalize wherever we need to.

Reply:
#744